microsoft-edge/devtools-guide-chromium/network-console/network-console-tool.md

```diff
@@ -89,6 +89,7 @@ See also:
 
 
 <!-- ====================================================================== -->
-<!-- ## See also -->
+## See also
 
+* [View formatted JSON](../json-viewer/json-viewer.md).
```

Comment:
```suggestion
* [View formatted JSON](../json-viewer/json-viewer.md)
```
I use period when there's info after the closing parentheses.  If this link were surrounded by verbose links, I'd maybe put a period.